security/firely-auth/firely-auth-endpoints.rst

```diff
@@ -117,3 +117,31 @@ Known Limitations
 
 * In Firely Auth no Backchannel Authentication Endpoint is available, therefore Client Initiated Backchannel Authentication (CIBA) requests are not supported. For more information, see `Duende Documentation - Client Initiated Backchannel Authentication (CIBA) <https://docs.duendesoftware.com/identityserver/v6/reference/endpoints/ciba/>`_.
 * A Device Authorization Flow is not supported by SMART on FHIR. Therefore it is not available in Firely Auth. For more information, see `Duende Documentation - Device Authorization Endpoint <https://docs.duendesoftware.com/identityserver/v6/reference/endpoints/device_authorization/>`_.
+
+Liveness and readiness
+----------------------
+
+It can be useful to check whether Firely Auth is still up and running, and ready to handle requests. Either just for notification, or for automatic failover.
+A prominent use case is to start dependent services only after Firely Auth is up and running, e.g. in a docker-compose or in a Helm chart.
+
+Firely Auth provides two endpoints, for different purposes:
+
+* ``GET <base>/$liveness``
+* ``GET <base>/$readiness``
+
+These align - intentionally - with the use of liveness and readiness probes in Kubernetes.
```

Comment:
Maybe good to provide a link to the kubernetes docs here

Reply:
done